payments-core/src/test/java/com/stripe/android/view/BecsDebitBanksTest.kt

```diff
@@ -16,12 +16,11 @@ class BecsDebitBanksTest {
             ApplicationProvider.getApplicationContext<Context>()
         ).banks
         assertThat(banks)
-            .hasSize(117)
+            .hasSize(121)
```

Comment:
Why did the number of banks change? Where did that come from?

Reply:
Yes, there were a few removed, but more added:
https://stripe.slack.com/archives/C032JPNDTHU/p1645797485901249